scripts/tests/test_suites.sh

```diff
@@ -18,7 +18,11 @@
 
 set -e
 
-declare -a test_array="($(find src/app/tests/suites -type f -name "*.yaml" -exec basename {} .yaml \;))"
+declare -a test_array=("Test_3_1_1"
```

Comment:
We should not be hardcoding the list of tests here.  People will be adding more tests and we want that to be as low-touch as possible.  My suggestion had been to place the TV yamls in a separate sub-directory of `src/app/tests/suites` and exclude that from the `find` command here (I can help with this part as desired).